File: JAVA/Program/Ram.java
public class Ram extends Komponen {
  private int kapasitasGB;
  private String ddr;

  public Ram() {
  }

  public Ram(int kapasitasGB, String ddr, String merk, String nama) {
    super(merk, nama);
    this.kapasitasGB = kapasitasGB;
    this.ddr = ddr;
  }

  public void setKapasitasGB(int kapasitasGB) {
    this.kapasitasGB = kapasitasGB;
  }

  public void setDdr(String ddr) {
    this.ddr = ddr;
  }

  public int getKapasitasGB() {
    return this.kapasitasGB;
  }

  public String getDdr() {
    return this.ddr;
  }
}

File: PYTHON/Program/Ram.py
from Komponen import Komponen

class Ram(Komponen):
  def __init__(self, kapasitasGB=0, ddr="", merk="", nama=""):
    super().__init__(merk, nama)
    self.kapasitasGB = kapasitasGB
    self.ddr = ddr

  def set_kapasitasGB(self, kapasitasGB):
    self.kapasitasGB = kapasitasGB

  def set_ddr(self, ddr):
    self.ddr = ddr

  def get_kapasitasGB(self):
    return self.kapasitasGB

  def get_ddr(self):
    return self.ddr
